package.json

```diff
@@ -33,6 +33,7 @@
     "fs-extra": "^11.1.1",
     "jspsych": "^7.2.3",
     "lodash": "^4.17.19",
+    "prop-types": "^15.8.1",
```

Comment:
This is in place of typescript?

Reply:
Right, I guess the react team moved prop types into its own library at some point. If/when Honeycomb becomes a separate library I think we'll move to typescript but I don't want to add the burden on new developers here